min-database-indexing.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: Xata 3-Minute Read: basics of database indexing 
```

Comment:
I don't know if we need to call out that it's a 3 min read. If it's going to be a series on a specific DB topic, why not just name the series or take a similar approach that Tudor did with the postgres search content

Reply:
Ah, just read the intro. I see what you're going for. My only suggestion would be to come up with something a bit more catchy. `Xata bytes` or something along those lines. It'd be cool to tie them to quick videos too, but that feels out of scope

Reply:
hah! yeah, let's roll with it if you like it too